composer.json

```diff
@@ -51,17 +51,11 @@
 		]
 	},
 	"scripts": {
-		"cs": [
-			"phpcs src/* tests/* --standard=phpcs.xml -sp"
-		],
 		"test": [
 			"@validate --no-interaction",
 			"covers-validator",
+			"phpcs -p -s",
 			"phpunit"
-		],
-		"ci": [
-			"@test",
-			"@cs"
 		]
```

Comment:
Why did you change this? I find it helpful to have the commands as they where, and this is what is what is done in a lot of WMDE components, so I'd be kinda surprised if there is no ci command.